components/match2/wikis/rainbow_six/match_summary.lua

```diff
@@ -307,45 +307,6 @@ function MapVeto:create()
 end
 
 
--- MVP Class
-local MVP = Class.new(
-	function(self)
-		self.root = mw.html.create('div'):addClass('brkts-popup-footer'):addClass('brkts-popup-mvp')
-		self.players = {}
-		self.points = nil
-	end
-)
-
-function MVP:addPlayer(player)
-	if not Logic.isEmpty(player) then
-		table.insert(self.players, player)
-	end
-	return self
-end
-
-function MVP:setPoints(points)
-	if Logic.isNumeric(points) then
-		self.points = points
-	end
-	return self
-end
```

Comment:
Points handling was removed from R6 and not added to commons. Would cause lua error on line `mvp:setPoints(mvpData.points)` in R6 custom.

Reply:
added to /Base